publiccode.yml

```
@@ -0,0 +1,155 @@
+
+      For virtual classrooms, BigBlueButton goes further — empowering educators
+      with built-in tools for active learning and live analytics to ensure
+      every student learns.
+    documentation: https://docs.bigbluebutton.org/
+    apiDocumentation: https://docs.bigbluebutton.org/development/api/
+    features:
+      - Record of sessions as a single .mp4 file
+      - Moderator role to manage users and session
+      - Built in whiteboard based on tldraw
+      - Localized into more than 65 languages
+      - Moodle, the worlds most popular LMS, built BigBlueButton into the core 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)